test/conftest.py

```diff
@@ -85,3 +86,16 @@ def get_file_paths(self, number):
             return file_list
 
     return _DataServe()
+
+
+@pytest.fixture(autouse=True)
+def qsettings_tmp_path(tmp_path):
```

Comment:
can you move this fixture after line 20 and put a big code comment clearly explaining this fixture is applied to every test?
Something like
```
##############################################
# FIXTURES AUTOMATICALLY APPLIED TO EVERY TEST
##############################################
```
This is to draw attention to the developer so that all "dark-magic" is revealed

Reply:
good idea